rotkehlchen/inquirer.py

```diff
@@ -342,7 +367,18 @@ def find_price(
             if cache is not None:
                 return cache.price
 
-        return instance._query_oracle_instances(from_asset=from_asset, to_asset=to_asset)
+        oracle_price = instance._query_oracle_instances(from_asset=from_asset, to_asset=to_asset)
+        if oracle_price == Price(ZERO):
```

Comment:
Why are you doing this? So if I understand correctly if no oracle has a price you are trying to find both asset usd prices and then use that to calculate their price in relation to each other?

Reply:
I still don't understand why you are adding this here. Everything we do is in usd all over the app. Conversions happen later in the frontend.

Reply:
You are right. When I was implementing this I got a conversion error for not finding the price due to a search for UNI-V2 token to EUR (might be because it was not fully functional yet and I didn't notice until the end). I've removed it, tested again and works fine. I was doing something wrong probably. Will push after fixing the other comments